content/documentation/Ox64/Software/Flashing.adoc

```diff
@@ -186,92 +186,88 @@ NOTE: Each time you open a new terminal window you will need to re-run `. ~/ox64
 
 * https://openbouffalo.org/static-assets/bldevcube/BouffaloLabDevCube-v1.8.3.zip
 * https://hachyderm.io/@mkroman/110787218805897192[] > https://pub.rwx.im/~mk/bouffalolab/BouffaloLabDevCube-v1.8.3.zip
 * https://we.tl/t-eJWShQJ4iF
 * https://cdn.discordapp.com/attachments/771032441971802142/1145565853962735639/BouffaloLabDevCube-v1.8.3.zip
 
-Verify that your copy of `BouffaloLabDevCube-v1.8.3.zip` matches the hashes below.
+Verify that your copy of `BouffaloLabDevCube-v1.8.3.zip` matches the hashes below:
 
 * SHA1: `0f2619e87d946f936f63ae97b0efd674357b1166`
 * SHA256: `e6e6db316359da40d29971a1889d41c9e97d5b1ff1a8636e9e6960b6ff960913`
 
 == Flashing the Ox64
 
-Put the Ox64 into programming mode.
+Put the Ox64 into programming mode:
 
 * Press the BOOT button
 * Apply power or re-plug the USB cable
 * Release the BOOT button
 
 === CLI flashing method
 
-Set up some environment variables to save typing them out later.
+Set up some environment variables to save typing them out later:
 
  PORT=/dev/ttyACM1 # or /dev/ttyUSB0, this will depend on which serial adapter you use
  BAUD=230400       # safe value for macOS, if using Linux set to 2000000 for faster flashing
 
-Change directory to the location of your image files.
+Change directory to the location of your image files:
 
  cd ~/ox64/openbouffalo/firmware # if you downloaded pre-built images
  # or
  cd ~/ox64/buildroot/output/images # if you built your own images
 
-Finally, flash the Ox64. If you created a **combined image** then run the command below.
+Finally, flash the Ox64. If you created a **combined image** then run the command below:
 
- bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x0 \
-               --firmware bl808-combined.bin --single
+ bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x0 --firmware bl808-combined.bin --single
 
-Otherwise, run the following commands.
+Otherwise, run the following commands:
 
- bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x0 \
-               --firmware m0_lowload_bl808_m0.bin --single
+ bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x0 --firmware m0_lowload_bl808_m0.bin --single
 
- bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x100000 \
-               --firmware d0_lowload_bl808_d0.bin --single
+ bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x100000 --firmware d0_lowload_bl808_d0.bin --single
 
- bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x800000 \
-               --firmware bl808-firmware.bin --single
+ bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x800000 --firmware bl808-firmware.bin --single
```

Comment:
I think the formatting is better with the explicit line wrap, so that it's clear that we're still typing the same (long) command.

Reply:
i followed the tip as per https://github.com/pine64/website/pull/35#issuecomment-1937817461

Reply:
That was referring to text paragraphs, not code listings.

In this case there's a difference in how it's rendered, not just how it appears in the source document.

Reply:
i tryed to revert this revision, but I can't do it

Reply:
You can copy/paste:

```
Finally, flash the Ox64. If you created a **combined image** then run the command below:

 bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x0 \
               --firmware bl808-combined.bin --single

Otherwise, run the following commands:

 bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x0 \
               --firmware m0_lowload_bl808_m0.bin --single

 bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x100000 \
               --firmware d0_lowload_bl808_d0.bin --single

 bflb-iot-tool --chipname bl808 --interface uart --port $PORT --baudrate $BAUD --addr 0x800000 \
               --firmware bl808-firmware.bin --single
```

Reply:
reverted